decidim-blogs/app/helpers/decidim/blogs/admin/posts_helper.rb

```diff
@@ -33,6 +33,17 @@ def post_author_select_field(form, name, _options = {})
 
           form.select(name, select_options)
         end
+
+        def publish_data(published_at)
+          data = {}
+          if published_at > Time.current
+            data[:icon] = icon("clock", aria_label: "Not published yet", role: "img")
+            data[:popup] = "Not published yet"
+          else
+            data[:popup] = "Published"
```

Comment:
I believe this case is no longer needed as we only display the icon and popup when it is not yet published. So maybe we can remove the `else` clause?

At least I cannot see this message in the UI.